docs/source/cn/math_ops.py

```diff
@@ -3,41 +3,1196 @@
+    oneflow.asinh,
+    r"""arcsinh(x) -> Tensor
+    
+    返回一个包含 :attr:`x` 中每个元素的反双曲正弦的新 tensor。
+
+    公式为：
+
+    .. math::
+        \text{out}_{i} = \sinh^{-1}(\text{input}_{i})
+
+    参数：
+        **x** (Tensor): 输入张量
+
+    关键词参数：
+        **out** (Tensor, optional): 输出张量
+
+    返回值：
+        oneflow.Tensor: 结果张量
+
+    示例：
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+
+        >>> input = flow.tensor([2, 3, 4], dtype=flow.float32)
+        >>> output = flow.asinh(input)
+        >>> output.shape
+        oneflow.Size([3])
+        >>> output
+        tensor([1.4436, 1.8184, 2.0947], dtype=oneflow.float32)
+
+        >>> input1 = flow.tensor([[-1, 0, -0.4], [5, 7, 0.8]], dtype=flow.float32)
+        >>> output1 = input1.asinh()
+        >>> output1.shape
+        oneflow.Size([2, 3])
+        >>> output1
+        tensor([[-0.8814,  0.0000, -0.3900],
+                [ 2.3124,  2.6441,  0.7327]], dtype=oneflow.float32)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.atan,
+    r"""arctan(x) -> Tensor
+
+    返回一个包含 :attr:`x` 中所有元素的反正切的新 tensor。
+
+    公式为：
+
+    .. math::
+        \text{out}_{i} = \tan^{-1}(\text{input}_{i})
+
+    参数：
+        **x** (Tensor): 输入张量
+
+    返回值：
+        oneflow.Tensor: 结果张量
+
+    示例：
+
+    .. code-block:: python
+    
+        >>> import oneflow as flow
+
+        >>> input = flow.tensor([0.5, 0.6, 0.7], dtype=flow.float32)
+        >>> output = flow.atan(input)
+        >>> output.shape
+        oneflow.Size([3])
+        
+    """,
+)
+
+reset_docstr(
+    oneflow.ceil,
+    r"""ceil(x) -> Tensor
+    
+    返回一个新的 tensor，tensor 中元素为大于或等于 :attr:`x` 中元素的最小整数。
+
+    公式为： 
+
+    .. math::
+        \text{out}_{i} = \left\lceil \text{input}_{i} \right\rceil = \left\lfloor \text{input}_{i} \right\rfloor + 1
+
+    参数：
+        **x** (oneflow.Tensor): 张量
+
+    返回值：
+        oneflow.Tensor: 结果张量
+
+    示例： 
+
+
+    .. code-block:: python 
+        
+        >>> import oneflow as flow
+
+        >>> x = flow.tensor([0.1, -2, 3.4], dtype=flow.float32)
+        >>> y = flow.ceil(x)
+        >>> y.shape
+        oneflow.Size([3])
+        >>> y
+        tensor([ 1., -2.,  4.], dtype=oneflow.float32)
+        >>> x = flow.tensor([[2.5, 4.6, 0.6],[7.8, 8.3, 9.2]], dtype=flow.float32)
+        >>> y = x.ceil()
+        >>> y.shape
+        oneflow.Size([2, 3])
+        >>> y
+        tensor([[ 3.,  5.,  1.],
+                [ 8.,  9., 10.]], dtype=oneflow.float32)
+        >>> x = flow.tensor([[[2.2, 4.4, 6.5],[7.1, 8.2, 9.3]],[[10.6,11.2,12.2],[13.5,14.8,15.9]]], dtype=flow.float32)
+        >>> y = flow.ceil(x)
+        >>> y.shape
+        oneflow.Size([2, 2, 3])
+        >>> y
+        tensor([[[ 3.,  5.,  7.],
+                 [ 8.,  9., 10.]],
+        <BLANKLINE>
+                [[11., 12., 13.],
+                 [14., 15., 16.]]], dtype=oneflow.float32)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.log1p,
+    r"""log1p(x) -> Tensor
+    
+    返回一个新的 tensor，其自然对数的公式为 (1 + x)。
```

Comment:
```suggestion
    返回一个新的 tensor，计算自然对数，公式为：
```

…… 这个已经改过一次了，有双方沟通理解偏差的问题……互相思考下下次怎么避免。